index.d.ts

```diff
@@ -1316,7 +1394,7 @@ declare namespace dashjs {
 
         getDashMetrics(): DashMetrics;
 
-        getQualityFor(type: MediaType): number;
+        getQualityFor(type: MediaType): QualityInfo;
```

Comment:
This only returns a number

Reply:
Nope, not what I see / get it is an quality object back...but this is fine and even better, as there are all information that I need. Index requires another call / request to get the correct quality information :) Maybe not intended, but a good from my point of view.

`console.log('getQualityFor("video")', this.dashjsplayerInstance.playerInstance.getQualityFor("video"));`

![image](https://github.com/Dash-Industry-Forum/dash.js/assets/7759961/a5a55727-6db6-4ebd-bcca-8bd9f40f5865)

Reply:
Is this based on the AdaptationSet Switching branch? Because for development it should be a number